platform/web/export/export_plugin.cpp

```diff
@@ -170,6 +170,9 @@ void EditorExportPlatformWeb::_fix_html(Vector<uint8_t> &p_html, const Ref<Edito
 	replaces["$GODOT_HEAD_INCLUDE"] = head_include + custom_head_include;
 	replaces["$GODOT_CONFIG"] = str_config;
 	replaces["$GODOT_SPLASH_COLOR"] = "#" + Color(GLOBAL_GET("application/boot_splash/bg_color")).to_html(false);
+	replaces["$GODOT_SPLASH_SHOW"] = GLOBAL_GET("application/boot_splash/show_image") ? "show-img" : "no-show-img";
+	replaces["$GODOT_SPLASH_FULLSIZE"] = GLOBAL_GET("application/boot_splash/fullsize") ? "fullsize" : "no-fullsize";
+	replaces["$GODOT_SPLASH_FILTER"] = GLOBAL_GET("application/boot_splash/use_filter") ? "linear-filter" : "nearest-neighbor";
```

Comment:
I think using the setting names is a good idea! How would you feel about adding classes in both cases (i.e. appending `_true` or `_false` to the class name)? I think it's less clunky and error-prone when creating custom HTML shells, since fewer assumptions have to be made in the custom CSS.

Reply:
@adamscott I've just pushed my suggestion, I'm more than happy to use your approach if you prefer it though. I'll squash the commits into one as soon as you think the PR is good to go!